### ISSUE TITLE
html `lang` goes `undefined` :walking: 

### ISSUE THREAD

**Issue by dargmuesli** [harlan-zw/nuxt-seo-kit#87]
### Describe the bug

When adding `@nuxtseo/module` AND using `useServerHeadSafe` to set the html's language, the language goes `undefined` in client side hydration :walking: 

Using `useHeadSafe` instead "fixes" the issue, removing `@nuxtseo/module` form the module array does too.

I would not expect this behavior. Am I doing something wrong or is this a good bug report? :smile: 

### Reproduction

https://stackblitz.com/edit/github-dpzmzy?file=nuxt.config.ts

### System / Nuxt Info

```Shell
------------------------------
- Operating System: Linux
- Node Version:     v16.20.0
- Nuxt Version:     3.7.0
- CLI Version:      3.7.2
- Nitro Version:    2.6.2
- Package Manager:  npm@9.4.2
- Builder:          -
- User Config:      modules
- Runtime Modules:  @nuxtseo/module@2.0.0-beta.19
- Build Modules:    -
------------------------------
```

**Comment by harlan-zw** [comment]
Thanks for the issue, looks to be fixed :) https://stackblitz.com/edit/github-dpzmzy-rtx8ry?file=app.vue

(also the nuxt-og-image bug)

**Comment by dargmuesli** [comment]
Perfect, thank you very much! :heart: :rocket:

**Comment by dargmuesli** [comment]
Hmm, what about the `title`? It becomes `null` on client hydration if it's set using `useServerHeadSafe`: https://stackblitz.com/edit/github-dpzmzy-ex5zxf?file=app.vue
Using `useHeadSafe` keeps the title intact.

**Comment by harlan-zw** [comment]
Thanks for the continued effort in helping me debug this, this seems to be fixed with Unhead 1.5.1.

The issue is basically that the client-side doesn't know what the original title is but is rendering a title template.
